### PR TITLE
Development

### PR DESCRIPTION
**Pull request by AlexSCorey**
I am pretty much done.

### CODE REVIEW

File: src/RandomColor.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function getRandomColor () {
```
Comment (cndreisbach) [inline src/RandomColor.js:1]: This is the coolest thing. You came up with a random color generator! The ability to come up with an idea outside of what you've seen before and try it out is the essence of programming.

File: src/Alien.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  draw () {
+    context.fillStyle = this.color
+    context.fillRect(this.center.x, this.center.y, this.size.x, this.size.y)
+  }
+  update () {
+    this.center.y += this.speed
+    if (this.center.y >= 500) {
+      this.game.gameOver = true
+    }
+    this.game.bullets.forEach((bullet, bulletIndex) => {
```
Comment (cndreisbach) [inline src/Alien.js:27]: I dig this section -- I like how you're checking for collisions and also updating the number of aliens and checking for the winner at once.

**Review by cndreisbach — APPROVED** [review]
I know this isn't exactly where you wanted it, but I'm really happy about it. You tried several new things and put a loop inside a loop, which isn't something most new programmers know they can do. This is good progress, and I'm looking forward to what you put together this weekend.

**Comment by AlexSCorey** [comment]
Thanks for the kind words.

**Pull request merged**